CODE_OF_CONDUCT.md

```diff
@@ -21,10 +21,11 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a professional setting
+* Use of unacceptable commit text or username is also prohibited.
```

Comment:
Line #24 : "Use of unacceptable commit text or username is also prohibited."

As someone who is **new** to the experience of commenting on a GitHub project, can someone clarify what this rule means? I am hoping that I am not confusing **username** with the @mention feature on GitHub. Thank you in return for helping my understanding of this matter.